### ISSUE TITLE
Testing Search Results Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4636]
# Page to Test:

Search Results - https://openlibrary.org/search?q=The+Crucible&mode=ebooks&has_fulltext=true

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by bpmcneilly** [comment]
@nynaalekhya - I think you expressed interest in testing this page, how's it going?

**Comment by Romi-776** [comment]
@bpmcneilly, I checked the search result of the page who's link is given above. 

I found out that on some pages there are small images of books listed below this line "n editions in n languages -- n previewable". These images of books have an empty alt attribute and doesn't have a title attribute.

**Example image:**
![Screenshot from 2021-03-05 08-23-17](https://user-images.githubusercontent.com/53464933/110061101-f7ba3080-7d8c-11eb-9f9e-ce04e28b2c10.png)

_These 9 images of the books doesn't have a title and have an empty alt attirbute._

**So, what should I do now in order to solve this?**

**Comment by Romi-776** [comment]
@bpmcneilly, I checked the search result of the page who's link is given above. 

I found out that on some pages there are small images of books listed below this line "n editions in n languages -- n previewable". These images of books have an empty alt attribute and doesn't have a title attribute.

**Example image:**
![Screenshot from 2021-03-05 08-23-17](https://user-images.githubusercontent.com/53464933/110061101-f7ba3080-7d8c-11eb-9f9e-ce04e28b2c10.png)

_These 9 images of the books doesn't have a title and have an empty alt attirbute._

**So, what should I do now in order to solve this?**

**Comment by nynaalekhya** [comment]
@Romi-776,
I'm working on this issue.

**Comment by nynaalekhya** [comment]
![wave_books](https://user-images.githubusercontent.com/50094420/110109466-574f1680-7d62-11eb-91af-387ebf3d419a.JPG)
There are nine img without alt attribute in preview covers section
